pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.480.3</version>
+    <version>1.599</version>
```

Comment:
Really? can you not live with even the most recent LTS?

I would think you could even go as far back as 1.565.1